frontend/lib/evictionfree/declaration-builder/routes.tsx

```diff
@@ -112,21 +131,25 @@ const EfLandlordMailingAddress = MiddleProgressStep((props) => (
     }
   >
     <p>
-      We'll use this information to send your declaration via certified mail for
-      free.
+      <Trans>
+        We'll use this information to send your hardship declaration form via
+        certified mail for free.
+      </Trans>
     </p>
   </LandlordMailingAddress>
 ));
 
 const EfOutsideNewYork: React.FC<ProgressStepProps> = (props) => (
   <Page
-    title="You don't live in New York"
+    title={li18n._(t`You don't live in New York`)}
     withHeading="big"
     className="content"
   >
     <p>
-      Unfortunately, this tool is currently only available to individuals who
-      live in the state of New York.
+      <Trans>
+        Unfortunately, this tool is currently only available to individuals who
+        live in the state of New York.
+      </Trans>
```

Comment:
FYI, this is text I just added this morning, and it was not part of the priority guide or otherwise "agreed upon" by anyone other than me.  We might want to make sure at least one other person is OK with it before translating it, but it's up to you!  (alternatively YOU could be that one other person, haha.)

Reply:
hahah ok good to know. I would consider this sort of an "Error message" as it's not something we'd be showing a regular user of the tool. Since we aren't getting external review from other error messages/pages, I'm ok to just move forward with this.